intake/tests/test_top_level.py

```diff
@@ -45,7 +45,7 @@ def tmp_path_catalog():
 
 
 def test_autoregister_open():
-    assert hasattr(intake, 'open_csv')
+    assert intake.open_csv
```

Comment:
What is the `assert` mean to verify?

Reply:
Not sure why I changed this one.
The test is to make sure that the open_csv function exists, given that the csv driver should have been auto registered.